lib/Test/Nginx/Util.pm

```diff
@@ -918,6 +918,12 @@ http {
     default_type text/plain;
     keepalive_timeout  68;
 
+    client_body_temp_path "$ServRoot/client_body_temp";
+    proxy_temp_path "$ServRoot/proxy_temp";
+    fastcgi_temp_path "$ServRoot/fastcgi_temp";
+    scgi_temp_path "$ServRoot/scgi_temp";
+    uwsgi_temp_path "$ServRoot/uwsgi_temp";
```

Comment:
@jirutka I suggest

1. you use your own build of nginx or builds that are not doing clever things with those temp directory paths, or
2. Use `--- main_config` to specify those settings yourself in your test cases.